.github/workflows/deploy.yml

```
@@ -5,13 +5,18 @@ on:
       - completed
+    branches:
+      - 'refs/tags/**' # Only trigger for tag-based workflows (releases)
   workflow_dispatch:
   release:
     types: [published]
 
 jobs:
   deploy:
-    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name != 'workflow_run' }}
+    if: |
+      github.event_name == 'workflow_dispatch' || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)